astroquery/esa/hubble/core.py

```diff
@@ -541,6 +540,43 @@ def query_target(self, name, *, filename=None, output_format='votable',
 
         return table
 
+    def query_tap(self, query, async_job=False, output_file=None,
+                  output_format="votable", verbose=False):
+        """Launches a synchronous or asynchronous job to query the HST tap
+
+                Parameters
+                ----------
+                query : str, mandatory
+                    query (adql) to be executed
+                async_job : bool, optional, default 'False'
+                    executes the query (job) in asynchronous/synchronous mode (default
+                    synchronous)
+                output_file : str, optional, default None
+                    file name where the results are saved if dumpToFile is True.
+                    If this parameter is not provided, the jobid is used instead
+                output_format : str, optional, default 'votable'
+                    results format
+                verbose : bool, optional, default 'False'
+                    flag to display information about the process
+
+                Returns
+                -------
+                A table object
+                """
+        if async_job:
+            job = self._tap.launch_job_async(query=query,
+                                             output_file=output_file,
+                                             output_format=output_format,
+                                             verbose=verbose,
+                                             dump_to_file=output_file is not None)
+        else:
+            job = self._tap.launch_job(query=query, output_file=output_file,
+                                       output_format=output_format,
+                                       verbose=verbose,
+                                       dump_to_file=output_file is not None)
+        table = job.get_results()
+        return table
+
     def query_hst_tap(self, query, async_job=False, output_file=None,
```

Comment:
This deprecation needs to be mentioned in the changelog, preferably as a separate entry.

Reply:
Deprecation included in the changelog. I have also ordered the different entries in the appropriate categories.